pkg/validation/validators/validators_test.go

```diff
@@ -71,6 +71,14 @@ func TestFinalNewline(t *testing.T) {
 		{"x\r", "false", "", errors.New("No final newline expected")},
 		{"x\r\n", "true", "", nil},
 		{"x\r\n", "false", "", errors.New("No final newline expected")},
+
+		// multiple newlines
+		{"x\n\n", "true", "", errors.New("Multiple newlines at end of file")},
+		{"x\r\r", "true", "", errors.New("Multiple newlines at end of file")},
+		{"x\r\n\r\n", "true", "", errors.New("Multiple newlines at end of file")},
+		{"x\n\n", "true", "lf", errors.New("Multiple newlines at end of file")},
+		{"x\r\r", "true", "cr", errors.New("Multiple newlines at end of file")},
+		{"x\r\n\r\n", "true", "crlf", errors.New("Multiple newlines at end of file")},
```

Comment:
This should be the only tests included in this PR to handle multiple newlines EOF.